how-to-guides/validator-node.md

```diff
@@ -181,6 +181,12 @@ Now, connect to the network of your choice.
 
 You have the following option of connecting to list of networks shown below:
 
+| Name                                                              | Type          | Chain ID    | CLI Usage               | 
+|-------------------------------------------------------------------|---------------|-------------|-------------------------|
+| [Celestia](https://docs.celestia.org/how-to-guides/mainnet)       | Mainnet Beta  | `celestia`  | `--chain-id celestia`   | 
+| [Mocha](https://docs.celestia.org/how-to-guides/mocha-testnet)    | Testnet       | `mocha-4`   | `--chain-id mocha-4`    | 
```

Comment:
let's use the constants here from https://github.com/celestiaorg/docs/blob/main/.vitepress/constants/constants.js to define the chain IDs!

Reply:
example: https://github.com/celestiaorg/docs/blob/46403eb64c90eb08e2af68150b417986ac2f180b/how-to-guides/celestia-node-troubleshooting.md?plain=1#L45

Reply:
also use relative links for the internal links